awslambdaric/bootstrap.py

```diff
@@ -257,7 +294,10 @@ def __init__(self, log_sink):
 
     def emit(self, record):
         msg = self.format(record)
-        self.log_sink.log(msg)
+
+        self.log_sink.log(
+            msg, frame_type=getattr(record, "_frame_type", _FRAME_TYPES.get(()))
```

Comment:
1. Should `_FRAME_TYPES.get(())` be replaced with just `None` ?
2. Nit: Is there any value for attribute lookup for non-JSON logging? If no, would be cleaner to avoid doing it